core/src/toga/style/pack.py

```diff
@@ -295,29 +295,42 @@ def apply(self, prop: str, value: object) -> None:
                 # so perform a refresh.
                 self._applicator.refresh()
 
-    def layout(self, node: Node, viewport: Any) -> None:
+    def layout(self, viewport: Any, _deprecated_usage=None) -> None:
+        ######################################################################
+        # 2024-12: Backwards compatibility for Travertino 0.3.0
+        ######################################################################
+
+        if _deprecated_usage is not None:
+            # Was called with (self, viewport)
+            viewport = _deprecated_usage
```

Comment:
There is presumably a change needed in Travertino itself to replace the call to `layout()` when the node is refreshed?

Reply:
Yes, it's one call in `Node.refresh()`. Simply *changing* it will crash and burn in the event that anyone still using Toga 0.4.8 updates to Travertino 0.4.0, so I put a backwards-compatibility check in https://github.com/beeware/travertino/pull/244